src/order/filter/pointwise.lean

```diff
@@ -5,179 +5,172 @@ Authors: Zhouhang Zhou
 -/
 import algebra.pointwise
 import order.filter.basic
+
 /-!
-# Pointwise operations on filters.
+# Pointwise operations on filters
+
+This file defines pointwise operations on filters. This is useful because usual filter operations
+distribute over pointwise operations. For example,
+* `(f₁ * f₂).map m  = f₁.map m * f₂.map m`
+* `𝓝 (x * y) = 𝓝 x * 𝓝 y`
+
+## Main declarations
+
+* `0 : filter α`: Principal filter at `0 : α`.
```

Comment:
Maybe worth something like
```suggestion
* `0 : filter α`(`filter.has_zero`): Principal filter at `0 : α`.
```
so that this is clickable in doc-gen